src/content/docs/usage/workspaces.mdx

```diff
@@ -105,11 +105,21 @@ Out of the box, Daytona can be configured to open your Workspace in the followin
     *   WebStorm
 *   Terminal SSH
 
+### Set the Default IDE
+You can set the default IDE used by Daytona.
+After setting the default IDE, it will be opened automatically and connected to your Workspace when using the `daytona open` command.
+
+__Prerequisites__
+*   A working installation of Daytona.
+
+__Procedure__
 1.  Execute the following command to enter the IDE selection terminal UI:
     ```shell
     daytona ide
     ```
 2.  Select your preferred IDE from the list presented.
 
-Your preferred IDE will be opened automatically and connected to your Workspace when using the `daytona open` command.
-
+:::note
+When using the `daytona open` command with `VS Code - Browser` as your default IDE, Daytona will set up an instance of OpenVSCode Server on your local machine.
```

Comment:
`code`

and `Daytona will install OpenVSCode Server inside your project, forward the appropriate port to your local machine and open the browser automatically`.